R/makeNeighbourhoods.R

```diff
@@ -101,30 +144,37 @@ makeNeighbourhoods <- function(x, prop=0.1, k=21, refined=TRUE, seed=42, reduced
 }
 
 #' @import igraph
-.sample_vertices <- function(graph, prop, return.vertices=FALSE, seed=42){
+.sample_vertices <- function(graph, prop, seed=42){
```

Comment:
This function needs to return either just the vertices or a list(vertices, vertex list) to prevent it break downstream functions.

Reply:
Is this really necessary if I use it just to call the initial random sample and then I compute the vertex list outside of the conditional statement? Are there other functions outside of `makeNeighbourhoods` that use `.sample_vertices`?

Reply:
Yes, it is used to populate the `@neighbourhoodIndex` slot that is used for the spatial FDR correction where the weighting uses the distance to the kth NN for the index vertex.